test/e2e/framework/framework.go

```diff
@@ -81,7 +81,8 @@ type Framework struct {
 }
 
 func newFramework(kubeconfig string, qps float32, burst int) (*Framework, error) {
-	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	logger := logrus.New()
```

Comment:
```suggestion
	logger = runtime.NewLoggerWithSource("framework")
```

Just to be consistent.